canisters/management/canister_management.ts

```diff
@@ -89,6 +94,28 @@ export const UpdateSettingsArgs = Record({
 });
 export type UpdateSettingsArgs = typeof UpdateSettingsArgs.tsType;
 
+export const UploadChunkArgs = Record({
+    canister_id: CanisterId,
```

Comment:
Please match the `ic.did` file exactly and make `CanisterId` `Principal` for consistency.

Reply:
Yeah, I'll create an issue for this.

Reply:
Can you link to that issue here?